src/main/java/com/wavefront/integrations/FDBMetricsReporter.java

```diff
@@ -253,6 +258,26 @@ private void createTailer(File logFile) {
                     tailer.stop();
                 }
             }
+
+            /*  
+            **  This method fetches the fdbtailer version number,
+            **  creates a custom metric and pushes it to the wavefront for monitoring.
+            */
+            private void sendFDBTailerVersionMetric() throws Exception {
+                try {
+                    String metricName = "fdbtailer.version";
```

Comment:
Change this so that it uses the SERVICE_NAME variable and concatenates it with the `".version"`.

Reply:
done